R/utils.R

```diff
@@ -203,12 +203,23 @@ replace_plus <- function(x){
 #                       check status                       ####
 #                                                          #
 ############################################################
+
+status_url <- function() {
+  "https://api.openaq.org/status"
+}
+
 get_status <- function(){
-  client <- crul::HttpClient$new(url = "https://api.openaq.org/status")
-  status <- client$get()
-  status <- suppressMessages(status$parse())
-  status <- jsonlite::fromJSON(status)
-  return(status$results$healthStatus)
+  client <- crul::HttpClient$new(url = status_url())
+  status <- try(client$retry("get"), silent = TRUE)
```

Comment:
this looks good to me, except that this doesn't discern between crul failing for the reason you are expecting, for the webmockr stub, vs. another reason.

Reply:
Right, it'd be clever to write if status != 200 or so I suppose.

Reply:
@sckott can you please have a look at the new diff?

Reply:
yes

Reply:
looks good